extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -105,14 +105,42 @@ export class SKURecommendationPage extends MigrationWizardPage {
 
 		const rbg = this.view!.modelBuilder.radioCardGroup();
 		rbg.component().cards = [];
+		rbg.component().orientation = azdata.Orientation.Vertical;
+		rbg.component().iconHeight = '30px';
+		rbg.component().iconWidth = '30px';
 
 		products.forEach((product) => {
 			const imagePath = path.resolve(this.migrationStateModel.getExtensionPath(), 'media', product.icon ?? 'ads.svg');
 
+			const descriptions: azdata.RadioCardDescription[] = [
+				{
+					textValue: product.name,
+					linkDisplayValue: 'Learn more',
+					displayLinkCodicon: true,
+					textStyles: {
+						'font-size': '1rem',
+						'font-weight': 550,
+					},
+					linkCodiconStyles: {
+						'font-size': '1em',
+						'color': 'royalblue'
+					}
+				},
+				{
+					textValue: '9 databases will be migrated',
```

Comment:
assume you will update this with actual values and localize the string properly.

Reply:
Yep, nothing in the extension side is set in stone yet